cs6232-g4/View/LoginForm.cs

```diff
@@ -110,5 +111,18 @@ private void TextPassword_Click(object sender, EventArgs e)
         {
             this.PasswordErrorLabel.Text = "";
         }
+
+        /// <summary>
+        /// Clears the textbox values
+        /// </summary>
+        public void Logout()
+        {
```

Comment:
this logout isn't part of the search feature, please revet it

Reply:
I cannot have the user logout without this method.

Reply:
We already have a logout link and logic implemented in the main form. So here's the problem in detail:

1- this code is a new duplicate logic which is already handled in DashboardForm.cs
2- there's no need to log users out in search furniture. If we do, then we have to duplicate the logic in all the features. So you don't have to worry about that, it's already done

Reply:
Addressed.